docs/guide/topics-cache.md

```diff
@@ -16,7 +16,7 @@ return [
 ];
 ```
 
-If you only use the redis cache, you can also configure the parameters of the connection within the
+If you use redis only as a cache, you can also configure the parameters of the connection within the
```

Comment:
should stay as is. It means the you only use the cache class and not e.g. session or AR.